ckanext/dalrrd_emc_dcpr/logic/action/dcpr.py

```diff
@@ -75,6 +75,7 @@ def dcpr_request_create(context, data_dict):
 
     try:
         model.Session.add(request)
+        model.repo.commit()
```

Comment:
This change seems totally unrelated to this PR. I guess you took it from one of the pending PRs. Please revert this, as I don't want to mix backend and frontend changes in this issue